Automattic/pocket-casts-ios
# SSO: hide change password/email for SSO logins

| 📘 Project: #381 |
|:---:|

The current endpoint doesn't support changing password/email for SSO login. This PR hides this option in case the user is logged in with either Google or Apple.

## To test

2. Go to `podcasts` scheme and change Build Configuration to `Staging`
3. On the app, go to Settings > Beta Features > enable `signInWithApple`
4. Login with username/password
4. Go to Profile > Tap the bubble account
4. ✅ You should see change password and change email options
4. Log out
4. Login with Google
4. Go to Profile > Tap the bubble account
4. ✅ You should **not** see change password and change email options
4. Log out
4. Login with Apple
4. Go to Profile > Tap the bubble account
4. ✅ You should **not** see change password and change email options

## Checklist
